mpi-proxy-split/lower-half/gethostbyname-static/Makefile

```diff
@@ -1,3 +1,4 @@
+GCC=gcc
```

Comment:
I suggest simply:  `CC=gcc` and continuing to use `CC` everywhere below.  This makes the change local.

Let's also add a small comment above it:
```
# This is needed for a statically linked MANA lower-half, in order to permit the statically linked
# code to use 'gethostbyname()'  We require 'gcc' here for this hack.
# When the MANA lower half is dynamically linked, all of this will go away.
```